### PR TITLE
qnamaker:build support --qnaConfig

### LINKED ISSUE #751: bf qnamaker:build should support --qnaconfig
**Is your feature request related to a problem? Please describe.**
luis:build --luconfig allows you to capture parameters in a generated file and gives more control.  qnamaker supports the same parameters and has the same needs, but does not support --qnaconfig equivalent.  

**Describe the solution you'd like**
Add --qnaconfig to qnamaker:build.

[enhancement]

### PR DESCRIPTION
**Pull request by feich-ms**
fix #751 to support --qnaConfig to configure the cli paramters. It also support to auto-extract subscriptionKey set by bf config:set:qnamaker.

### CODE REVIEW

File: packages/qnamaker/src/commands/qnamaker/build.ts [hunk 1]
```diff
@@ -27,59 +30,89 @@ export default class QnamakerBuild extends Command {
...
       flags.stdin = await this.readStdin()
 
-      if (!flags.stdin && !flags.in) {
+      if (!flags.stdin && !inVal && files.length === 0) {
         throw new CLIError('Missing input. Please use stdin or pass a file or folder location with --in flag')
       }
 
-      if (flags.dialog && flags.dialog !== recognizerType.MULTILANGUAGE && flags.dialog !== recognizerType.CROSSTRAINED) {
+      if (!subscriptionKey) {
+        throw new CLIError('Missing QnAMaker subscription key. Please pass subscription key with --subscriptionKey flag or specify via bf config:set:qnamaker.')
```
Comment (vishwacsena) [inline packages/qnamaker/src/commands/qnamaker/build.ts:82]: bf config:set:qnamaker --subscriptionKey <your subscription key>

File: packages/qnamaker/src/commands/qnamaker/build.ts [hunk 2]
```diff
@@ -27,59 +30,89 @@ export default class QnamakerBuild extends Command {
...
+      }
+
+      if (!botName) {
+        throw new CLIError('Missing bot name. Please pass bot name with --botName flag.')
```
Comment (vishwacsena) [inline packages/qnamaker/src/commands/qnamaker/build.ts:86]: Or specify via --qnaConfig

Comment (feich-ms) [reply packages/qnamaker/src/commands/qnamaker/build.ts]: resolved

File: packages/qnamaker/src/commands/qnamaker/build.ts (hunk 1 above)
Comment (feich-ms) [reply packages/qnamaker/src/commands/qnamaker/build.ts]: resolved

**Pull request merged**